test_scripts/Defects/8_0/3788/001_allow_RC_permission_in_case_App_is_registered_with_MEDIA_appHMIType.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3788
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL assigned RC permissions for the App in case:
+-- - 'REMOTE_CONTROL' appHMIType is assigned for App by policy
+-- - app is registered with 'MEDIA' appHMIType
+--
+-- In case:
+-- 1. RC functionality is allowed App by policy
+-- 2. 'REMOTE_CONTROL' appHMIType is allowed by policy for App
+-- 3. App is registered with 'MEDIA' appHMIType
+-- 4. App is activated
+-- 5. App sends valid SetInteriorVehicleData request
+-- SDL does:
+-- - allow App's remote-control RPCs (success:true, "SUCCESS")
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Defects/8_0/3788/common_3788')
+
+--[[ Test Configuration ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local mod = "RADIO"
```

Comment:
Maybe rename variable for better clarifying: 
mod => modTyp 
or
 mod => moduleType
```suggestion
local modTyp = "RADIO"
```

Reply:
same in other files